.orchestra/config/components/mass-testing.yml

```diff
@@ -0,0 +1,98 @@
+#@ load("@ytt:overlay", "overlay")
+#@ load("/lib/fn-args.lib.yml", "mandatory")
+
+#@yaml/text-templated-strings
+---
+#@ def mass_testing_config(
+#@      report_name=mandatory,
+#@      download_specs=[],
+#@      pre_configure=None,
+#@      post_install=None,
+#@  ):
+builds:
+  default:
+    build_dependencies:
+    - ninja
+    - revng-qa
+    - revng-c
+    - sqlite
+
+    configure: |
+      rm -rf "$BUILD_DIR/build"
+      mkdir -p "$BUILD_DIR/"{inputs,build}
+      cd "$BUILD_DIR"
+
+      BASE_DIR="$ORCHESTRA_DOTDIR/support/mass-testing"
+      export S3_ENDPOINT="$MASS_TESTING_S3_ENDPOINT"
+
+      (@- if pre_configure: @)
+      (@=- pre_configure @)
+      (@- end @)
+
+      "$BASE_DIR/download_binaries" \
+        "$BUILD_DIR/inputs" \
+        (@= " ".join(download_specs) @)
+
+      "$BASE_DIR/generate_meta" meta.yml
```

Comment:
What does this command do?

Reply:
Takes the file `root/share/revng/mass-testing-meta.yml` and adds the `notes:` field which generates the `revng version: ...` string. (Will probably inline it in the `mass-testing.yml` file)

Reply:
Add some comments

Reply:
Done